### PR TITLE
bump cna to 0.9.0

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

File: deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1171,11 +1171,11 @@ spec:
                     valueFrom:
                       fieldRef:
                         fieldPath: metadata.namespace
-                  image: rthallisey/hyperconverged-cluster-operator:latest
+                  image: docker.io/rthallisey/hyperconverged-cluster-operator:latest
                   imagePullPolicy: Always
-                  name: hyperconverged-cluster-operator
+                  name: rthallisey/hyperconverged-cluster-operator
```
Comment (phoracek) [inline deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: This change appeared after ./hack/build-manifests.sh. Is that intended @rthallisey ?

Comment (rthallisey) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: That doesn't look right.   Something might be wrong in `build-manifests.sh`

Comment (phoracek) [reply deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml]: Should be fixed now

File: hack/defaults [hunk 2]
```diff
@@ -6,7 +6,7 @@ WAIT_TIMEOUT="${WAIT_TIMEOUT:-300s}"
 # Use upstream manifests with downstream containers
 KUBEVIRT_MANIFEST_VERSION="v0.17.0"
 CDI_MANIFEST_VERSION="v1.6.0"
-NETWORK_ADDONS_MANIFEST_VERSION="0.7.0"
+NETWORK_ADDONS_MANIFEST_VERSION="0.9.0"
```
Comment (SchSeba) [inline hack/defaults]: should be need to add also the nmstate to this file as NETWORK_ADDONS_NMSTATE_STATE_HANDLER_IMAGE ?

Comment (phoracek) [reply hack/defaults]: Oh, nice catch! You are right

**Pull request merged**